pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: test
-version: 0.12.7-dev
```

Comment:
i'd guess that you'll want to leave their dependency entries alone before PR'ing into master unless there is an actual problem with 1 of the deps